pkg/applier/status.go

```diff
@@ -118,12 +126,14 @@ func (m ObjectStatusMap) Log(logger infofLogger) {
 	}
 }
 
-func writeStatus(w io.Writer, status interface{ String() string }, ids []core.ID) {
+func writeStatus(w io.Writer, status interface{ String() string }, ids []core.ID) error {
+	var err error
 	if len(ids) == 0 {
-		fmt.Fprintf(w, "%s (%d)", status, len(ids))
+		_, err = fmt.Fprintf(w, "%s (%d)", status, len(ids))
 	} else {
-		fmt.Fprintf(w, "%s (%d): [%s]", status, len(ids), joinIDs(commaSpaceDelimiter, ids...))
+		_, err = fmt.Fprintf(w, "%s (%d): [%s]", status, len(ids), joinIDs(commaSpaceDelimiter, ids...))
 	}
+	return err
```

Comment:
nit: can we move the panic inside the function instead of returning an error and then panic outside?

Reply:
done